core/unit_test/TestTeamCombinedReducers.hpp

```diff
@@ -0,0 +1,459 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <Kokkos_Core.hpp>
+
+#include <iostream>
+#include <limits>
```

Comment:
```suggestion
#include <gtest/gtest.h>
```

Reply:
Changed in [2b7a12b](https://github.com/kokkos/kokkos/pull/5727/commits/2b7a12b285fce804e0f299e22aa59ce332397810).